lib/localization.js

```diff
@@ -56,7 +54,7 @@ const _localizeDefaultObjectID = function (change, locale) {
 const _localizeEntityType = function (change, locale) {
     if (change.entity) {
         try {
-            const labelI18nKey = _getLabelI18nKeyOnEntity(change.entity);
+            const labelI18nKey = _getLabelI18nKeyOnEntity(change.serviceEntity);
```

Comment:
This is the actual fix. The other changes are merely cosmetics

Reply:
👍